build/vsts_build.yaml

```diff
@@ -441,7 +441,12 @@ phases:
     demands: 
         - DotNetFramework
         - msbuild
-
+    matrix:
+      IsDesktop:
+        SkipCoreAssemblies: "true"
+      IsCore:
```

Comment:
Core

Reply:
Leaving it this way, consistent with metadata.